nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/cloud.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function () {
+  this.Given('provider "$provider" exists in providers', (provider) => {
+    fixtures.providers.create({
```

Comment:
Missing `return` here and there's no need to the additional braces. We could just have
```
(provider) => fixtures.providers.create({
      'entity-type': 'nuxeoOAuth2ServiceProvider',
      serviceName: provider,
    })
```